lib/rack/ecg/check/sequel_connection.rb

```diff
@@ -0,0 +1,39 @@
+module Rack
+  class ECG
+    module Check
+      class SequelConnection
```

Comment:
Yep, will do, wanted to get some early feedback, hence the fact that this is WIP 😄